Remotion/Data/DomainObjects/Data.DomainObjects.csproj

```diff
@@ -28,7 +28,7 @@
     <PackageReference Include="Remotion.Linq.EagerFetching" Version="2.2.0" />
     <PackageReference Include="Remotion.Linq.SqlBackend" Version="2.1.1" />
     <PackageReference Include="Remotion.TypePipe" Version="3.0.0" />
-    <PackageReference Include="System.Data.SqlClient" Version="4.8.2" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />
```

Comment:
If I am not mistaken there are still two other usages of the package that should also be upgraded.